mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -53,7 +53,7 @@ def test_acq_skip():
     assert_equal(len(raw.times), 17000)
     annotations = raw.annotations
     assert_equal(len(annotations), 3)  # there are 3 skips
-    assert_allclose(annotations.onset, [2, 7, 11])
+    assert_allclose(annotations.onset, [14, 19, 23])
```

Comment:
This suggests that there are some backward-incompatible changes or bugfixes. Can you update `whats_new.rst` to describe them?